common/util/autoware_utils/include/autoware_utils/ros/debug_traits.hpp

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef AUTOWARE_UTILS__ROS__DEBUG_TRAITS_HPP_
+#define AUTOWARE_UTILS__ROS__DEBUG_TRAITS_HPP_
+
+#include <type_traits>
+
+#include "autoware_debug_msgs/msg/bool_stamped.hpp"
+#include "autoware_debug_msgs/msg/float32_multi_array_stamped.hpp"
+#include "autoware_debug_msgs/msg/float32_stamped.hpp"
+#include "autoware_debug_msgs/msg/float64_multi_array_stamped.hpp"
+#include "autoware_debug_msgs/msg/float64_stamped.hpp"
+#include "autoware_debug_msgs/msg/int32_multi_array_stamped.hpp"
+#include "autoware_debug_msgs/msg/int32_stamped.hpp"
+#include "autoware_debug_msgs/msg/int64_multi_array_stamped.hpp"
+#include "autoware_debug_msgs/msg/int64_stamped.hpp"
+#include "autoware_debug_msgs/msg/string_stamped.hpp"
+
+namespace debug_traits
```

Comment:
@wep21 Thank you for creating this library. I think it's better to add namespace of package name(autoware_utils).

Reply:
Thanks for your comment. I fixed it.